main.go

```diff
@@ -84,6 +85,10 @@ func zoneList() []*ftp.Entry {
 		log.Fatal(err)
 	}
 
+	sort.SliceStable(entries, func(i, j int) bool {
```

Comment:
I don't know that we care whether we use `SliceStable` vs just `Slice` here do we?

Reply:
After looking at the docs a little more, it seems like keeping the same order if they are the same is fine.  Any reason to use `slice` instead?

Reply:
Also stable sorts usually use about double the memory...but yeah...performance prob doesn’t really matter in this case. I was just curious if there was a reason to use stable sort vs sort in this case.

Reply:
Ah yeah good to keep in mind.  I am going to merge this as in this case it doesn't really matter - but will keep this in mind for future golang stuff :)